src/orion/algo/mofa/sampler.py

```diff
@@ -0,0 +1,119 @@
+"""
+MOFA Sampler module
+"""
+
+# from skopt.sampler import Lhs
+# from skopt.space import Space as SkSpace
+
+import logging
+
+import numpy as np
+from scipy.stats.qmc import LatinHypercube
+
+logger = logging.getLogger(__name__)
+
+
+class Sampler:
```

Comment:
A class with only static methods is a bit of a code smell. Is the class even needed at this point?
You could probably remove the class, and use the module instead as the base, e.g.:

From this:
```python
from orion.algo.mofa.sampler import Sampler
foo = Sampler.generate_trials(...)
```
to this:
```python
from orion.algo.mofa import sampler
foo = sampler.generate_trials(...)
```

Reply:
Addressed in https://github.com/Epistimio/orion/pull/864/commits/8e12929ff7d85949bc4b8743f4e6baa105f73482